Jenkinsfile

```diff
@@ -38,6 +38,35 @@ pipeline {
                 }
             }
         }
+
+        stage('Checkout mm') {
+            steps {
+                dir('mm') {
+                    git url: 'https://github.com/zeldaret/mm.git'
+                }
+            }
+        }
+        stage('Set up mm') {
+            steps {
+                dir('mm') {
+                    sh 'cp /usr/local/etc/roms/mm.us.rev1.z64 baserom.z64'
+
+                    // Identical to `make setup` except for copying our newer ZAPD.out into oot
+                    sh 'git submodule update --init --recursive'
+                    sh 'python3 -m pip install -r requirements.txt'
```

Comment:
We shouldn't be installing any dependencies at build time. What's missing that requires this?

Reply:
I copy-pasted that from MM's makefile. From the setup step to be precise.
https://github.com/zeldaret/mm/blob/aae6d9d3644f0792a27b3c67758345c6d13e909b/Makefile#L196

I guess I should remove this, right?

Reply:
Ah okay. My bad for missing it there then. Yeah I think it can be removed